sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -304,30 +300,28 @@ public static void ProcessMultiplePartitionEvents([EventHubTrigger(TestHubName)]
 
         private Tuple<JobHost, IHost> BuildHost<T>()
         {
-            JobHost jobHost = null;
-
-            var config = new ConfigurationBuilder()
-                .AddEnvironmentVariables()
-                .AddTestSettings()
-                .Build();
-
-            const string connectionName = "AzureWebJobsTestHubConnection";
-            string connection = config.GetConnectionStringOrSetting(connectionName);
-            Assert.True(!string.IsNullOrEmpty(connection), $"Required test connection string '{connectionName}' is missing.");
-
+            JobHost jobHost;
             IHost host = new HostBuilder()
+                .ConfigureAppConfiguration(builder =>
+                {
+                    builder.AddInMemoryCollection(new Dictionary<string, string>()
+                    {
+                        { "AzureWebJobsStorage", TestEnvironment.StorageAccountConnectionString }
+                    });
+                })
                 .ConfigureDefaultTestHost<T>(b =>
                 {
                     b.AddEventHubs(options =>
                     {
                         // TODO: alternative?
                         //options.EventProcessorOptions.EnableReceiverRuntimeMetric = true;
-                        options.AddSender(TestHubName, connection);
-                        options.AddReceiver(TestHubName, connection);
+                        options.AddSender(TestHubName, TestEnvironment.EventHubsNamespaceConnectionString);
```

Comment:
At some point in the future, we should probably swap over to `producer` and `consumer` terminology to align with the T2 SDK and the Event Hubs service.

Reply:
There is a push to keep APIs compatible as much as possible but we can evaluate how much do we care about these particular methods.